src/search/landmarks/landmark_factory_h_m.cc

```diff
@@ -629,8 +632,8 @@ void LandmarkFactoryHM::calc_achievers(const TaskProxy &task_proxy) {
             FluentSet post = get_operator_postcondition(variables.size(), operators[op_id]);
             FluentSet pre = get_operator_precondition(operators[op_id]);
             size_t j;
-            for (j = 0; j < lmn->facts.size(); ++j) {
-                const FactPair &lm_fact = lmn->facts[j];
+            for (j = 0; j < landmark.facts.size(); ++j) {
```

Comment:
I guess but not by simply replacing it due to the check if something was done with the FactPair (see line 663, where it is checked if j is equal to landmark.facts.size()). I don't think it's hard to change this, but for that I'd have to look deeper into what is going on here, and when I do, I suspect that more than just this will change. Therefore, I'm leaning towards saying this is out of scope of the issue.

Reply:
I agree but this is the kind of change that looks simple and then introduces some bug anyway, so I wouldn't want to change it without running a (small) experiment, and I'd rather not run another experiment.